utam-compiler/src/test/resources/validate/custom_element/nestedElementsForList.json

```diff
@@ -4,13 +4,14 @@
       "name": "test",
       "type": "my/page/type",
       "selector": {
-        "css": "css1"
+        "css": "css",
+        "returnAll": true
       },
-      "elements" : [
+      "elements": [
         {
-          "name": "nested",
-          "selector": {
-            "css": "css2"
+          "name" : "nested",
```

Comment:
What's our consistency here for JSON formatting? Do we expect a space before the colon or not? I don't particularly care, but let's at least make an attempt to be consistent within a single file. :D

Reply:
we have spotless to take care of that